ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -18,6 +18,7 @@ type structure_interface_alias = {
     "ip"      ? type_ip
     "netmask" : type_ip
     "broadcast" ? type_ip
+    "fqdn"    ? string
```

Comment:
Every interface has a fqdn in DNS. e.g. myhost-hb.dc1.quattor.org is an interface of myhost.dc1.quattor.org.